README.md

```diff
@@ -1,31 +1,32 @@
 Ansible Role: Hostname
 =========
-| Master branch | Developer branch | 
-|:---:|:---:|
-| Master branch: [![Build Status](https://travis-ci.org/insspb/ansible-role-hostname.svg?branch=master)](https://travis-ci.org/insspb/ansible-role-hostname) | Developer branch: [![Build Status](https://travis-ci.org/insspb/ansible-role-hostname.svg?branch=develop)](https://travis-ci.org/insspb/ansible-role-hostname) |
 
 Description
 ------------
 
-This role will set hostname on any target host. By default hostname is same as inventory_hostname, but can be set as different by using vars. Updates hosts file too.
+This role will set hostname on any target host.
+By default hostname is same as `inventory_hostname`,
+but can be set as different by using vars. Updates hosts file too.
 
 Requirements
 ------------
 
 No requiments.
 
 - CentOs 5 requires **python-simplejson** package installed for general Ansible work. 
-- Ubuntu 16.04 has no **python** package installed by default. It is reuired for general Ansible to work. 
+- Ubuntu 16.04 has no **python** package installed by default. It is required for general Ansible to work. 
 
 Role Variables
 --------------
 
-| Variable                     | Default                  | Explanation  |
-| -----------------------------|--------------------------| -----|
-| hostname_hosts_file_location | "/etc/hosts"             | Hosts file location. |
+| Variable                     | Default                  | Explanation                                                    |
+| ---------------------------- | ------------------------ | -------------------------------------------------------------- |
+| hostname_hosts_file_location | "/etc/hosts"             | Hosts file location.                                           |
 | hostname_hosts_backup        | False                    | You can make backup of hosts file before changes will be made. |
-| hostname_fqdn_full           | inventory_hostname | By default this role use **inventory_hostname** from inventory file to get host hostname, but you can set **hostname_fqdn_full** and **hostname_fqdn_short** to change this behavior.|
-| hostname_fqdn_short          | inventory_hostname_short | |
+| hostname_fqdn_full           | inventory_hostname       | By default this role use **inventory_hostname** from inventory file to get host hostname, but you can set **hostname_fqdn_full** and **hostname_fqdn_short** to change this behavior. |
+| hostname_fqdn_short          | inventory_hostname_short |                                                                |
+| hostname_hosts_ipv4_enabled  | true                     | When true, add an IPv3 entry to /etc/hosts                     |
```

Comment:
```suggestion
| hostname_hosts_ipv4_enabled  | true                     | When true, add an IPv4 entry to /etc/hosts                     |
```

Reply:
Good spot. Now that's an interesting protocol...